expan/core/early_stopping.py

```diff
@@ -125,6 +126,37 @@ def HDI_from_MCMC(posterior_samples, credible_mass=0.95):
     return (HDImin, HDImax)
 
 
+def compile_stan_model(model_file, distribution):
```

Comment:
It's not about size. This method does two things, load the precompiled model if existed or create the model. 
I would prefer to name it as ```action1_or_action2``` to be clearer what it does. What do you think?

Not sure about being "Pythonic", but this is a common practice in Java community.
e.g. 
[getOrDefault](https://google.github.io/guava/releases/21.0/api/docs/com/google/common/collect/ImmutableMap.html#getOrDefault-java.lang.Object-V-) in Google Guava
[getOrCreate](https://spark.apache.org/docs/2.0.1/api/java/org/apache/spark/sql/SparkSession.Builder.html#getOrCreate()) in Apache Spark.